cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -325,6 +325,7 @@ writeMarshalUnmarshalParams(Output& out, const ParamDeclList& params, const Oper
     }
 
     bool cpp11 = (typeCtx & TypeContextCpp11) != 0;
+    bool tuple = (typeCtx & TypeContextTuple) != 0;
```

Comment:
This is temporary - all this code needs to be cleaned up. A good start would be to remove all the C++98 code.